__tests__/utils/common_test.js

```diff
@@ -1,47 +1,80 @@
-import { showQuestion, getLink } from "../../utils/common";
+import { showQuestion, getLink, questionIsRelevant } from "../../utils/common";
 import questionsFixture from "../fixtures/questions";
 import multipleChoiceOptionsFixture from "../fixtures/multiple_choice_options";
 import questionDisplayLogicFixture from "../fixtures/question_display_logic";
+import eligibilityPathsFixture from "../fixtures/eligibilityPaths";
+
+describe("questionIsRelevant function", () => {
+  let reduxState;
+  beforeEach(() => {
+    reduxState = {
+      eligibilityPaths: eligibilityPathsFixture,
+      multipleChoiceOptions: multipleChoiceOptionsFixture
+    };
+  });
+
+  it("returns false if question is not relevant", () => {
+    const profileFilters = {
+      patronType: "p2"
```

Comment:
Using p1, p2 etc makes it harder to understand what the tests are doing without looking at the fixture.